src/Frame.jsx

```diff
@@ -94,7 +94,7 @@ export default class Frame extends Component {
       </Content>
     );
 
-    if (initialRender) {
+    if (initialRender && this.props.initialContent) {
```

Comment:
`initialContent` has a default value so this clause will always trigger on first render. What is this trying to solve?

Reply:
If you explicitly set the `initialContent` to '' then this will not be triggered.

The issue it solves is that Firefox, at least in my case, threw a security-related error when trying to open and write to the document.

Setting the initialContent to '' and the mountTarget to body did the trick.

Reply:
I think I'd like to understand how this error is happening are you able to create a codesanbox with reproduction steps?

Right now I'm not sure how this is working for you since if you reset `initialContent` and it tries to render it looks for a mount target either specified by the `mountTarget` prop or the first child inside body which would be undefined if you set `initialContent` to an empty string.